projects/plugins/jetpack/extensions/blocks/author-recommendation/style.scss

```diff
@@ -17,6 +17,12 @@
 
         img.site-icon {
             border-radius: 50%;
+			min-width: 36px;
+			max-height: 36px;
+		}
+
+		div {
+            min-width: 36px;
```

Comment:
```suggestion
            max-width: 36px;
            max-height: 36px;
        }
        
        div {
            max-width: 36px;
```

Reply:
Fixes the spacing and sets the logos to an even square. I think this is better for spacing in a table, otherwise the logos can push the text out like this:

<img width="318" alt="Markup 2023-05-18 at 15 54 31" src="https://github.com/Automattic/jetpack/assets/33258733/856c5cd9-51e0-4fdc-b19a-68c6f20921a8">

Also may want to apply this styling to the editor because it's a jungle in there haha:

<img width="1202" alt="Markup 2023-05-18 at 15 39 45" src="https://github.com/Automattic/jetpack/assets/33258733/c30ef65a-efb1-462c-b249-c22f83fac8f0">

Reply:
Nice catch. As discussed in the squad channel the site icon can come in sizes bigger than expected.
To handle this I committed the appropriate CSS rules to limit the size on the front-end.